### ISSUE TITLE
Doesn't Install via Unity Package Manager

### ISSUE THREAD

**Issue by simonpaulkeating** [soupday/cc_unity_tools_3D#26]
Using  Unity 2021.3.3f1 the package doenst install via the package manager with the following error:

[Package Manager Window] Cannot perform upm operation: Unable to add package [https://github.com/soupday/cc_unity_tools_3D.git]:
  No 'git' executable was found. Please install Git on your system then restart Unity and Unity Hub [NotFound].
UnityEditor.EditorApplication:Internal_CallUpdateFunctions ()

I have Git for Desktop installed in C:\Users\myName\AppData\Local\GitHubDesktop\GitHubDesktop.exe

I also have that path variable setup to C:\Users\myName\AppData\Local\GitHubDesktop is my environment variables

Restarted Unity many times but the process does not work. I've had git repositories import fine into Unity previously.

**Comment by Mateus2905** [comment]
Have you figured it out? I'm having the same problem

**Comment by liudger** [comment]
Did you install https://git-scm.com/download/win ?

**Comment by Mateus2905** [comment]
> Did you install https://git-scm.com/download/win ?

Yes I did. However, I was fortunately capable to install the package another way.
